frontendExternalSystem/src/main/java/com/virtuslab/gitmachete/frontend/externalSystem/project/MacheteProjectResolver.java

```diff
@@ -0,0 +1,55 @@
+package com.virtuslab.gitmachete.frontend.externalSystem.project;
```

Comment:
Hmm there's `externalSystem` (camel case) but also `datakeys`, `gitcore` (all lower case) etc. elsewhere

Reply:
If we want to stick to all lower case everywhere, pls add a Regexp rule

Reply:
okay so lets stick to lowercase 
This can be our oracle: https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names